### ISSUE TITLE
Ruby warnings of constant redefinition

### ISSUE THREAD

**Issue by ixti** [dry-rb/dry-core#11]
Pretty much subj:

```
/home/ixti/.gem/ruby/2.3.3/gems/dry-validation-0.10.3/lib/dry/validation/message_compiler.rb:13: warning: already initialized constant Dry::Validation::MessageCompiler::EMPTY_OPTS
/home/ixti/.gem/ruby/2.3.3/gems/dry-core-0.2.1/lib/dry/core/constants.rb:52: warning: previous definition of EMPTY_OPTS was here
```

Not a greatest issue, but still would be nice to get rid of those ;))

**Comment by flash-gordon** [comment]
Yeah, sorry about that, this will fixed in dry-validation 0.10.4, see https://github.com/dry-rb/dry-validation/issues/276

**Comment by ixti** [comment]
@flash-gordon 👍 Thanks!
